roles/version_update_single_node/tasks/update.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Get all available running VMs
+  scale_computing.hypercore.vm_info:
+  register: vms
+
+# -------------SHUTDOWN-ALL-RUNNING------------------
+
+- name: Show all running VMs
+  ansible.builtin.debug:
+    msg: "{{ item.vm_name }}"
+  when: item.power_state == 'started'
+  loop: "{{ vms.records }}"
+  register: running_vms
+
+- name: Shutdown running VMs
+  scale_computing.hypercore.vm_params:
+    vm_name: "{{ item.vm_name }}"
+    power_state: stop
+  when: item.power_state == 'started'
+  loop: "{{ vms.records }}"
+  register: vm_shutdown_result
+
+- name: Show shutdown results
+  ansible.builtin.debug:
+    var: vm_shutdown_result
+
+# ---------------------UPDATE-----------------------
+
+# - name: Update HC3
+#   scale_computing.hypercore.api:
+#     action: post
+#     endpoint: "/rest/v1/Update/{{ item.uuid }}/apply"
```

Comment:
This will be replaced with call of module Polona is writing.
 
No other comments.